tests/assert_test.expect

```diff
@@ -1,2 +1,2 @@
-x > 5 && xassert_msg("x is not greater than 5") (../src/main.xc:7)
+x > 5 && xassert_msg("x is not greater than 5") (../src/main.xc:8)
```

Comment:
Adding an extra comment line for the licence reference changes the expected output as the line number where this message originates is increased by one.